components/eam/bld/namelist_files/use_cases/2000_scream_hr.xml

```diff
@@ -80,13 +80,13 @@
 <use_hetfrz_classnuc>.false.</use_hetfrz_classnuc>
 <aerodep_flx_type>'CYCLICAL'</aerodep_flx_type>
 <aerodep_flx_datapath>atm/cam/chem/trop_mam/aero</aerodep_flx_datapath>
-<aerodep_flx_file nlev="72">mam4_0.9x1.2_L72_2000clim_c170323.nc</aerodep_flx_file>
```

Comment:
Again I wonder if replacing F2000 prescribed aero with F2010 is a good idea here. As an aside, I think these "non-dyamond2-scream-compsets" should probably just be changed to "dyamond1 compsets", so this particular change is probably fine, so long as we then change this to a 2010_scream_hr_dyamond1.xml compset and update forcings. At that point though, it probably makes more sense to keep this an F2000 and base a new dyamond1 compset on the F2010.

Reply:
Again reverting to the old input file for F2000 compsets. Switching DYAMOND1 to have its own use case based on 2010 is a good idea... for another PR at a later date.